### PR TITLE
add webhooks documentation

### PR DESCRIPTION
**Pull request by UziTech**
This is copied and changed from Gogs webhooks documentation

https://gogs.io/docs/features/webhook

### CODE REVIEW

File: content/doc/features/webhooks.en-us.md [hunk 1]
```diff
@@ -15,4 +15,87 @@ menu:
...
+### Event information
+
+Following shows an example of event information that will be sent by Gitea to Payload URL:
+
+
+```
+X-Github-Delivery: f6266f16-1bf3-46a5-9ea4-602e06ead473
+X-Github-Event: push
+X-Gogs-Delivery: f6266f16-1bf3-46a5-9ea4-602e06ead473
+X-Gogs-Event: push
```
Comment (lafriks) [inline content/doc/features/webhooks.en-us.md:29]: Please add here also:
```
X-Gitea-Delivery: f6266f16-1bf3-46a5-9ea4-602e06ead473
X-Gitea-Event: push
```

**Review by lafriks — CHANGES_REQUESTED** [review]

File: content/doc/features/webhooks.en-us.md (hunk 1 above)
Comment (UziTech) [reply content/doc/features/webhooks.en-us.md:29]: done

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
LGTM

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Pull request merged**